CHARTER.md

```diff
@@ -0,0 +1,232 @@
+# JSON Schema Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation. The JSON Schema project strives to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas we wish to give focus to, while secondary concerns are more likely to require the desire of the community to drive the work.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON data
+  - Semantic annotation of JSON data
+  - Extensibility
+- Critical tooling
+  - Creation of new
+  - Ingestion of existing
```

Comment:
What does ingestion mean? Other comments imply that it means "take over ownership of".

Reply:
Given this seems to be a contrivertial thing (https://github.com/openjs-foundation/cross-project-council/issues/1059 and two other comments on this line), I've removed the specifics, which we can debate as required.

Additionally, after [discussing the topic at our in person summit](https://github.com/orgs/json-schema-org/discussions/391), I'm more in favour of trying to support implementers as much as possible to keep a project alive. Our strength is in the ecosystem, and implementers feeling empowered will help this.

Resolving for now.